scipy/optimize/_linprog_highs.py

```diff
@@ -131,40 +132,54 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
         to be disabled.
     dual_feasibility_tolerance : double
         Dual feasibility tolerance.  Default is 1e-07.
-        This option is ignored when ``solver='ipm'``.
+        The minimum of this and ``primal_feasibility_tolerance``
+        is used for the feasibility tolerance when ``solver='ipm'``.
     dual_objective_value_upper_bound : double
         Upper bound on objective value for dual simplex:
         algorithm terminates if reached.  Default is the largest possible
         value for a ``double`` on the platform.
         When ``solver='ipm'`` this value is ignored.
+    ipm_optimality_tolerance : double
+        Optimality tolerance for ``solver='ipm'``.  Default is 1e-08.
+        Minimum possible value is 1e-12 and must be smaller than the largest
+        possible value for a ``double`` on the platform.
     message_level : int {0, 1, 2, 3, 4, 5, 6, 7}
         Verbosity level, corresponds to:
 
-            ``0``: No messages
+            ``0``: None
+                All messaging to stdout is supressed.
 
-            ``1``: Verbose messaging
+            ``1``: Verbose
+                Includes a once-per-iteration report on simplex/ipm
+                progress and information about each nonzero row and
+                column.
 
-            ``2``: Detailed messaging
+            ``2``: Detailed
+                Includes technical information about progress and
+                events in applying the simplex method.
 
-            ``3``: Verbose and Detailed messaging
+            ``3``: Verbose | Detailed
```

Comment:
```suggestion
            ``3``: Verbose and Detailed
```
"How is the bitwise or operation defined between two English words?" I understand this means that `3` = `1 | 2`, but that's not what is written. Since we're spelling out all the combinations as separate integer values, and since "Verbose" and "Detailed" are English words and not named constants representing integers `1` and `2`, I think it would be less confusing to revert to English "and" here and possibly eliminate the note below about it behaving like a bitmask.

Reply:
Yeah, that was not best way to write it.  I've changed everything to English descriptions